components/dictionary.txt

```diff
@@ -0,0 +1,310 @@
+al
+aes
+Alboukadel
+alevin
+alevinQC
+Alevin
+Alevin's
+AlJanahi
+Amezquita
+AML
+Anders
+Angerer
+AnnotationDbi
+AnnotationHub
+astrocytoma
+BAM
+Baran
+barcodes
+Becht
+bioconductor
+bioinformatic
+bioinformatics
+Biomedicine
+bioRxiv
+biospecimen
+BMC
+Bonferroni
+bp
+BSIDs
+Castelo
+CBs
+CCDL
+CCDL's
+cDNA
+CDKN
+centric
+cheatsheet
+cheatsheets
+clusterProfiler
+Cmd
+colData
+ComplexHeatmap
+concordantly
+conda
+CPMs
+csv
+Ctrl
+customizable
+customization
+customizations
+cytometry
+Danio
+Darmanis
+DataFrames
+dataset
+deconvolved
+DESeq
+DESeq2
+DESeqDataSet
+df
+DGE
+digoxin
+Digoxin
+DIPG
+displaystyle
+Dockerhub
+DocToc
+dplyr
+dropdown
+DropSeq
+dumpFeatures
+eigengene
+ENA
+Enricher
+Ensdb
+Ensembl
+ENSG
+Entrez
+et
+Ewels
+Exponentiate
+ExtractoR
+FAC
+FACS
+fastp
+fastq
+FASTQ
+FastQC
+FCS
+fgsea
+Figshare
+FN
+FPKM
+frac
+Freitag
+Gardeux
+GBM
+GC
+GeneCard
+GeneCards
+geo
+GG
+ggplot
+ggplots
+GitHub
+glioblastoma
+GRCh
+gridlines
+GSE
+GSEA
+GSMXXXXXXX
+GSVA
+Guinney
+gz
+Hadley
+Hänzelmann
+hardcoded
+Harenza
+HBC
+Healy
+Hematol
+Hemberg
+hexamer
+histologies
+Hm
+https
+interpretable
+interpretability
+Illumina
+imap
+immunophenotypic
+irectory
+jitter
+JSON
+Kassambara
+KEGG
+keytype
+keytypes
+Kiselev
+Korotkevich
+Khatri
+kmers
+Lafzi
+LFC
+Liberzon
+loadings
+lockfile
+log'd
+LowKit
+Luecken
+Lun
+lv
+LV
+LVs
+macrophage
+Malhotra
+McInnes
+medulloblastoma
+MEK
+Menyhárt
+mer
+misordered
+Mll
+MLL
+MOG
+Monti
+mRNA
+MSigDB
+mTOR
+MultiQC
+murine
+Muris
+musculus
+MYB
+MYCN
+myelination
+myelogenous
+myeloid
+NCI
+NES
+Neuro
+neuroblastoma
+Neuroblastoma
+NK
+nonamplified
+Nonamplified
+Novia
+NRAS
+NRASG
+octothorps
+OkabeIto
+oligodendrocyte
+oncogenesis
+oncoproteins
+Oncol
+OpenPBTA
+orking
+orthotopic
+overexpressed
+Patro
+Paugh
+PDX
+ped
+Phred
+Picelli
+PLIER
+PLoS
+plotly
+Plotly
+plotPCA
+PNOC
+pre
+preprint
+preprocess
+preprocessed
+Preprocesses
+preprocessor
+preranked
+proteasome
+Proteasome
+pseudocount
+QCs
+quant
+quantifications
+quasimapping
+RAS
+rds
+README
+Relevel
+replot
+reproducibility
+rerio
+rhelp
+rmd
+Rmd
+RPKMs
+rRNA
+Rscript
+RStudio
+RStudio’s
+Sca
+scater
+SCE
+scran
+scRNA
+Sergushichev
+setosa
+SingleCellExperiment
+SingleCellExperiments
+Sirota
+snakemake
+SNE
+Soneson
+SOX
+Srivastava
+SRA
+SRP
+SRR
+StackOverflow
+str
+stringr
+subfolder
+Subramanian
+subtype
+subtypes
+Sukhov
+Summarise
+Summarises
```

Comment:
Do we want to just pick a spelling here and change these to summarize in the text?
```suggestion
summarise
summarises
```